### PR TITLE
Akaya Telivigala: Version 1.001; ttfautohint (v1.8.3) added

### PR DESCRIPTION
**Pull request by yanone**
f149f0f: [gftools-packager] Akaya Telivigala: Version 1.001; ttfautohint (v1.8.3) added

* Akaya Telivigala Version 1.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/vaishnavimurthy/Akaya-Telivigala.git at commit https://github.com/vaishnavimurthy/Akaya-Telivigala/commit/db01c66d22057fd8f5e5d433d1ef3b37098ca209.

### CODE REVIEW

File: ofl/akayatelivigala/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  copyright: "Copyright 2015 The Akaya Telivigala Project Authors (https://github.com/vaishnavimurthy/Akaya-Telivigala)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "telugu"
+source {
+  repository_url: "https://github.com/vaishnavimurthy/Akaya-Telivigala.git"
+  commit: "db01c66d22057fd8f5e5d433d1ef3b37098ca209"
+}
```
Comment (m4rc1e) [inline ofl/akayatelivigala/METADATA.pb:22]: When using the packager, you want to include `--no-source`. @davelab6 do we know if this field is supported on the backend yet?

Comment (yanone) [reply ofl/akayatelivigala/METADATA.pb:22]: Want or have to? What does it do? And shall I change anything now for this PR?

Comment (m4rc1e) [reply ofl/akayatelivigala/METADATA.pb:22]: apologies but `have` makes more sense. Don't worry about changing anything, I'll take it out myself.

**Comment by m4rc1e** [comment]
Updated

## Akaya Telivigala: Version 1.001; ttfautohint (v1.8.3) added

---
 f149f0f: [gftools-packager] Akaya Telivigala: Version 1.001; ttfautohint (v1.8.3) added

* Akaya Telivigala Version 1.001; ttfautohint (v1.8.3) taken from the upstream repo https://github.com/vaishnavimurthy/Akaya-Telivigala.git at commit https://github.com/vaishnavimurthy/Akaya-Telivigala/commit/db01c66d22057fd8f5e5d433d1ef3b37098ca209.

**Pull request merged**